rpc/block_write_stream.go

```diff
@@ -173,7 +173,7 @@ func (s *blockWriteStream) makePacket() outboundPacket {
 	// gets unhappy unless we first align to a chunk boundary with a small packet.
 	// Otherwise it yells at us with "a partial chunk must be sent in an
 	// individual packet" or just complains about a corrupted block.
-	alignment := outboundChunkSize - (int(s.offset) % outboundChunkSize)
+	alignment := (outboundChunkSize - (int(s.offset) % outboundChunkSize)) % outboundChunkSize
```

Comment:
I think I see the problem now, ugh. An extra `%` just makes this line more confusing, though. I think something like this would scan better:

```
alignment := int(s.offset) % outboundChunkSize
if packetLength > outboundChunkSize && alignment != 0 {
    packetLength = outboundChunkSize - alignment
}
```

What do you think?